### ISSUE TITLE
Spree v3_0 support

### ISSUE THREAD

**Issue by vmatekole** [priviterag/spree_slider#36]
Hi, 

I am new to Spree and am using v3_0 as it comes out of the box with bootstrap support. When looking at the admin section, there is no slider option rendered, furthemore I cannot see any errors on the console and all migrations have been ran. Happy to fix this in a PR, if you can point me in the right direction. 

~victor

**Comment by JDutil** [comment]
I don't think this extension has been updated for 3.0. The deface override needs to be updated to add the link in the configuration menu:
https://github.com/priviterag/spree_slider/blob/master/app/overrides/spree/admin/shared/_configuration_menu/add_slides_to_admin_configurations_sidebar_menu.html.erb.deface

However I recommend you begin following https://github.com/spree-contrib/spree_slider as I believe work on this repo has moved there
